core/tx_pool.go

```diff
@@ -169,9 +169,9 @@ var DefaultTxPoolConfig = TxPoolConfig{
 	PriceLimit: 1,
 	PriceBump:  10,
 
-	AccountSlots: 32,
+	AccountSlots: 1,
```

Comment:
I think 1 might be too low. We should make an issue to re-think this later.

Reply:
https://github.com/dominant-strategies/go-quai/issues/881